Overview.md (#78)
AgregoreWeb/website: Overview.md

### Using local AI
Inorder to use the AI Chat, you need to have [ollama](https://ollama.com/) downloaded onto your computer. Once that is done you are able to use Agregores LLM Chat app! Go to Docs>Examples>LLM Chat and type in a conversation or question you have for the AI and within seconds the LLM Chat will have a response! 

### Built in AD Blockers
When you go on to a website in a browser, you usually get a lot of pop up ads and banner ads. This can be frustrating and overwhelming to see unwanted ads/information while your browsing the web. One of the benefits to Agregore is that it has it's own built in ad blocker! This means that when you use the browser, you won't see any ads and can just focus on what your searching!
### Sharing data locally without internet
If your sending a file to someone and it's inside a hyper url and are on the same wifi, that person will be able to load the file from you directly. This means that even if your wifi isnt connected to the internet, you or the person who wants the file, can still attain the file.

### Customizing using Web Extentions

> When you use Agregore you can customize how you want your browser to look like. This mean that when you go to a webpage, you can change how the site looks to fit your needs. Lets say you prefer to read headings or text that is purple. Well with Agregore you can do just that! Click on Agregores **Web Extention** (second button on the top right corner) and tada! The Webpage suddenly changes to the color code that you want! For more information about customizing Agregores color and style, visit **Agregore>Docs>Theme Builder**. Also you can add more extentions. Go to **Help>Open Extentions Folder** and you are able to add whatever other extetention files you want. Giving you more freedom to express your own style in your web browser!

Reviewer: This should be split into three sections:

1. Web Extensions: Show how you can "open the extensions folder" from `Help > Open Extensions Folder` and extract an extension to add to Agregore like [MetaMask](https://github.com/MetaMask/metamask-extension/releases). If you have trouble we can walk through it together
2. Custom Styling: Agregore lets you customize the color scheme to look however you want. Navigate to `agregore://settings` and set a custom background color (like red or something), show how the welcome page and any apps will now show that color too
3. Reader Mode Extension: Webpages can have a bunch of extra styles and headers that get in the way of reading them. Agregore has a built in "Reader mode" extension" that shows just the contents of a page with your configured color scheme

### Archive webpages with Webrecord:
When you archive a webpage, you save a copy of the site to your computer through the integrated Webrecorder extension. This can help keep your sites organized and keep things private until you want to share with others. If you would like to download Webrecorder click [here](https://webrecorder.net/archivewebpage/).

### Peer to Peer Webapps
Peer to peer webpages are applications that give users the ability to share data and communicate to each other instantly and directly. Some examples of peer to peer webpages on Agregore include:

**[Drag and Drop File Uploader](https://agregore.mauve.moe/docs/examples/drag-and-drop/)**: Take a file from Finder and drop it in and it will upload it to the peer to peer web!

> **[LLM App Generator](https://agregore.mauve.moe/docs/examples/llm-appgen/)**: For this Generator you need to have AI installed. Once AI has been installed this will use the built in AI support that will generate an app for you. You simply type in what you want the app to do or what you want it to look like and the app generator will create the app and publish it to the peer to peer web.

Reviewer: ```suggestion
**[LLM App Generator](https://agregore.mauve.moe/docs/examples/llm-appgen/)**: For this you need to have AI installed. This will use Agregore's built in AI support to generate an app for you. You simply type in what you want the app to do or what you want it to look like and the app generator will create the app and publish it to the peer to peer web.
```
